app/scripts/lib/createDupeReqFilterMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import log from 'loglevel';
+
+/**
+ * Returns a middleware that filters out requests already seen
+ *
+ * @returns {Function}
+ */
+export default function createDupeReqFilterMiddleware() {
+  const processedRequestId = [];
+  return function filterDuplicateRequestMiddleware(
+    /** @type {any} */ req,
+    /** @type {any} */ _,
```

Comment:
Nit: it might be useful to spell out what this is, so it's more clear why we're ignoring it. The leading `_` still shows we're not using it on purpose.
```suggestion
    /** @type {any} */ _res,
```